### PR TITLE
[3554][MIG] account_move_product: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3554](https://www.quartile.co/web#view_type=form&model=project.task&id=3554&active_id=3554&menu_id=)

### CODE REVIEW

File: account_move_product/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    product_id = fields.Many2one(
+        "product.product", related="invoice_line_ids.product_id", string="Product"
+    )
```
Comment (yostashiro) [inline account_move_product/models/account_move.py:12]: In 16.0, there is search on `line_ids` (not in 15.0) and I think we may just take that approach.  I think that should suffice.  Let's remove this field and just adjust the search view.  Please change the module name to `account_move_search_line` and propose the module in OCA as appropriate.

image

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Comment by yostashiro** [comment]
Please also switch the license to AGPL.

File: account_move_product/models/account_move.py (hunk 1 above)
Comment (kanda999) [reply account_move_product/models/account_move.py:12]: Now that this PR is trying to move to the **15.0** version, line_ids cannot be used?

Comment (AungKoKoLin1997) [reply account_move_product/models/account_move.py:12]: invoice_line_ids is also exit in v15.0 and we can also use for search. https://github.com/odoo/odoo/blob/4ee1def9c11021e1f692ab9cb769005aa2fbdbd0/addons/account/models/account_move.py#L316.

**Comment by kanda999** [comment]
This PR is followed up by https://github.com/qrtl/sst-oca/pull/33

**Pull request closed without merging**